app/models/micropost.rb

```diff
@@ -0,0 +1,5 @@
+class Micropost < ApplicationRecord
+	belongs_to :user
+	validates :content, length: { maximum: 140 }
+	presence: true
```

Comment:
Hi, we have an error here:
![Screenshot 2019-10-10 at 12 07 07 PM](https://user-images.githubusercontent.com/10922508/66563886-a9344280-eb56-11e9-9cf8-6535b63fc2ee.png)